src/utils/config/fetch.ts

```diff
@@ -24,6 +24,10 @@ import { getConfigOrThrow } from "./config";
 // Returns a fetch wrapped with timeout and retry logic
 //
 const API_TIMEOUT = getConfigOrThrow().CHECKOUT_API_TIMEOUT as Millisecond;
+const RETRY_NUMBERS_NORMAL = getConfigOrThrow()
+  .CHECKOUT_API_RETRY_NUMBERS_NORMAL as Millisecond;
+const EXPONENT = getConfigOrThrow()
+  .CHECKOUT_API_RETRY_NUMBERS_EXPONENT as Millisecond;
```

Comment:
```suggestion
const RETRY_NUMBERS_NORMAL = getConfigOrThrow()
  .CHECKOUT_API_RETRY_NUMBERS_NORMAL as number;
const EXPONENT = getConfigOrThrow()
  .CHECKOUT_API_RETRY_NUMBERS_EXPONENT as number;
```

These are respectively a _count_ and a _multiplier_, right? They're not expressed in milliseconds

Reply:
Done